src/scripts/transaction-utils.js

```diff
@@ -63,3 +63,20 @@ export const getCoin = transaction => {
     return transaction.value.amount
   }
 }
+
+// Required by cosmos-sdk/MsgMultiSend txs to fetch coins from transaction inputs[] or outputs[] arrays
```

Comment:
This states what it does not why.

```suggestion
// We currently don't support MultiCoin transactions in the design. For simplicity we display only the first outgoing or incomming denomination.
```